opensearch_dashboards.json

```diff
@@ -13,5 +13,7 @@
     "dataSourceManagement"
   ],
   "server": true,
-  "ui": true
+  "ui": true,
+  "supportedOSDataSourceVersions": ">=2.3.0",
```

Comment:
should this be 2.0.0? From the web site, we have notification plugin bundled since 2.0.0 

https://opensearch.org/docs/latest/install-and-configure/plugins/#bundled-plugins

Reply:
addressed